docs/general/faq.md

```diff
@@ -161,6 +161,21 @@ imposes economic costs on the operators of the bridge, and therefore makes it ec
 Bridge efforts are being worked on in concert with other projects in the ecosystem. Eventually,
 there will be bridges between Polkadot and most of the other major chains.
 
+### What is Polkadot's Transactions Per Second (TPS)?
+
+Polkadot's TPS is currently around the range of 1,000 TPS and is **anticipated** to range from
+[100,000 to 1,000,000 transactions per second.](https://polkadot.network/blog/polkadot-roadmap-roundup).
+This is primarily due to the pending [asynchronous backing upgrade](../learn/learn-async-backing).
+
+It is essential to realize that TPS is inherently a subjective measurement with numerous factors
+that can contribute to it. It's hard to gauge the usefulness of TPS in isolation, as it's highly
+dependent on what a transaction does for a particular network.
+
+Furthermore, there is no industry standard for measuring TPS non-biasedly.
+
+In the case of Polkadot, the TPS can be very high due to the parallelization of its network. Because
+it is effectively finalizing the state of many parachains at once.
```

Comment:
```suggestion
In the case of Polkadot, TPS can be high if we consider parallelization. Because Polkadot is sharded network,
it is effectively finalizing the state of many parachains at once.
```